feat: Document "Fix issues" button (#2116)
codacy/docs: docs/repositories/pull-requests.md

![Commits tab](images/pull-requests-tab-commits.png)

## Fixing issues automatically

!!! info "This section applies to GitHub repositories only"

> If Codacy detects code patterns that have suggested fixes, you can apply the fixes directly by clicking the **Fix issues** button that appears above the issue list. This action applies the suggested fixes and commits the changes to the pull request.

Reviewer: Same here. Does the following make sense?

```suggestion
If Codacy detects code patterns that have suggested fixes, you can apply those fixes directly by clicking the **Fix issues** button above the issue list. This action commits the suggested fixes to the pull request.
```
